### PR TITLE
fix(groups): remove excessive info from group listing imprint

### PR DESCRIPTION
**Pull request by hypeJunction**
groups

### CODE REVIEW

File: mod/groups/views/default/group/default.php [hunk 1]
```diff
@@ -15,10 +15,32 @@
...
+		'content' => elgg_echo('groups:closed'),
+		'class' => 'elgg-listing-group-membership elgg-state elgg-state-danger',
+	];
+}
+
 $icon = elgg_view_entity_icon($entity, 'small', $vars);
 
 $vars['content'] = $entity->briefdescription;
 $vars['handler'] = 'groups';
+$vars['byline'] = false;
```
Comment (jeabakker) [inline mod/groups/views/default/group/default.php]: not sure about this one

**Comment by jeabakker** [comment]
rest LGTM

File: mod/groups/views/default/group/default.php (hunk 1 above)
Comment (hypeJunction) [reply mod/groups/views/default/group/default.php]: I don't think owner info is important in the summary view

**Comment by hypeJunction** [comment]
Test failure is unrelated

File: mod/groups/views/default/group/default.php (hunk 1 above)
Comment (jeabakker) [reply mod/groups/views/default/group/default.php]: that's my point. I think owner information can be very useful in the summary/listing view ;)

Comment (jdalsem) [reply mod/groups/views/default/group/default.php]: talked with @jeabakker about this... he can live with it :)

**Pull request merged**